Doc/library/unittest.rst

```diff
@@ -1486,8 +1486,87 @@ Test cases
       .. versionadded:: 3.8
 
 
+.. class:: async_case.IsolatedAsyncioTestCase(methodName='runTest')
```

Comment:
The class is exported on the package level, e.g. `unittest.IsolatedAsyncioTestCase` works.
Please drop `async_case` prefix